### ISSUE TITLE
Duplicate outline won't appear instantly

### ISSUE THREAD

**Issue by sural98** [gantry/gantry5#741]
Version: dev-9f2931b5f / Released: August 19, 2015
Go to Outlines
Duplicate an outline
Duplicate won't appear instantly
You will need to make a hard refresh of the page or go via "Components/Gantry 5 Themes" to view the duplicated outline.
Saved outlines need also a page refresh to reflect the changes. Without refreshing it will load as if it is newly duplicated.
Another issue here is that drag-and-drop for duplicated layouts won't work and page will need a refresh to get it work again.

**Comment by mahagr** [comment]
@w00fz Looks like we need to update outlines page anyway.. I have already partially implemented this (by sending updated box back in JSON), but I'm not sure if I had it implemented for creation.

**Comment by w00fz** [comment]
I am a bit confused. It does appear instantly for me and the whole Outline page is refreshing after duplication, the new HTML is sent via Ajax response and that is what's getting used to refresh the page.
I am also able to normally drag & drop in Layout Manager.

I see you have mentioned a CI from Aug 19, we did have other issues concerning the Layout Manager that got fixed after that day, can you try a more recent CI Build?

**Comment by adi8i** [comment]
For me is working fine. Version: dev-0c6770253 / Released: August 24, 2015

**Comment by mahagr** [comment]
Ok, it's working for me again. And yes, it reloads the page again, too.

**Comment by sural98** [comment]
I have just installed "Version: dev-164d7c22d / Released: August 24, 2015".
The problem I had noticed yesterday was that nothing was changing in the address bar of the browser while loading the layouts but now address is pointing to the layout that is being loaded.
Anyway the issue is gone (expect forever:))

**Comment by sural98** [comment]
The problem is back!
The layout I changed yesterday is loaded today as if there were no changes.
Everything was OK only after a page refresh!
